.github/workflows/query-list.yml

```
@@ -29,21 +29,15 @@
~      uses: actions/setup-python@v3
       with:
         python-version: 3.8
     - name: Download CodeQL CLI
-      uses: dsaltares/fetch-gh-release-asset@aa37ae5c44d3c9820bc12fe675e8670ecd93bd1c
-      with:
-        repo: "github/codeql-cli-binaries"
-        version: "latest"
-        file: "codeql-linux64.zip"
-        token: ${{ secrets.GITHUB_TOKEN }}
+      uses: ./codeql/.github/actions/fetch-codeql
```

If this is a path within the submodule, shouldn't it be...
```suggestion
      uses: ./ql/.github/actions/fetch-codeql
```

---

We're grabbing it from https://github.com/github/codeql/blob/main/.github/actions/fetch-codeql/action.yml — unusually the `codeql` repo is checked out at `codeql`, since we also want to check out `codeql-go`.

I'll add a comment to explain so others hopefully won't get confused in the future, as this is a little surprising.

---

Ah, thanks.